benchmark/bench-report/bin/bench-runner.sh

```diff
@@ -436,15 +436,20 @@ run_reports() {
 
     for i in $1
     do
-        echo "Generating reports for ${i}..."
-        $prog \
+        res=$($prog \
             --benchmark $i \
             $(test "$USE_GAUGE" = 1 && echo "--use-gauge") \
             $(test "$GRAPH" = 1 && echo "--graphs") \
             $(test "$SORT_BY_NAME" = 1 && echo "--sort-by-name") \
             $(test -n "$BENCH_DIFF_STYLE" && echo "--diff-style $BENCH_DIFF_STYLE") \
             $(test -n "$BENCH_CUTOFF_PERCENT" && echo "--diff-cutoff-percent $BENCH_CUTOFF_PERCENT") \
-            --fields "$FIELDS"
+            --fields "$FIELDS")
+
+        if [ -n "$res" ]
```

Comment:
I usually use `test -n "$res"` rather than using `[`. I find `[` a bit weird, it is not some shell syntax, it is a command with the name `[`, you can type `which [` and see. `test` is more explicit.